### DISCUSSION TITLE
User Management Example: NextJS is throwing Hydration errors

### DISCUSSION THREAD

**Discussion by jbbae** [supabase/supabase#16908] [Contribute to Supabase]
# Bug report

## Describe the bug

I simply cloned "supabase", navigated to examples/user-management/nextjs-user-management, installed, added my keys, and ran "yarn run dev" as instructed, but I am getting the below error:

> Error: Hydration failed because the initial UI does not match what was rendered on the server.
> 
> Warning: Did not expect server HTML to contain a <div> in <div>.
> 
> See more info here: https://nextjs.org/docs/messages/react-hydration-error

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Cloned "supabase"
2. Navigated to examples/user-management/nextjs-user-management
3. yarn install
4. Added my Supabase keys to .env.local
5. Ran "yarn run dev"

## Expected behavior

No errors, since I've made 0 edits to the code other than adding my Supabase keys.

## Screenshots

![image](https://github.com/supabase/supabase/assets/12439968/c3d52513-b74f-483c-96c6-c9dc97d801fb)

## System information

- OS: Windows 11
- Browser: chrome
- Version of supabase-js: 2.26.0
- Version of Node.js: 18.15.0

**Comment by ramsrib** [comment]
I'm also facing the same issue.

**Comment by marcNY** [comment]
Same error

**Comment by gregnr** [comment]
Thanks for reporting @jbbae. Would love to see a PR to fix this!

**Comment by KSSaiTeja** [comment]
can i work on this issue..?

**Comment by danlzh** [comment]
Had no issue with the example. Maybe you should try Chrome Incognito mode, some browser extensions / experiments could cause the hydration error (see https://nextjs.org/docs/messages/react-hydration-error).

**Comment by Zydiag** [comment]
I was working on one of my next project and got the same issue on debugging I found that it was due to an chrome extension for video controlller. check the extension in your case.
I fixed my isssue so I will try to fix this one now.

**Comment by Zydiag** [comment]
actually I need to know the extension to reproduce the error @jbbae could you check your extensions
